docs/maps/maps-getting-started.asciidoc

```diff
@@ -166,13 +168,61 @@ Now that your map is complete, you'll want to save it so others can use it.
 . In the application toolbar, click *Save*.
 . Enter `Tutorial web logs map` for the title.
 . Click *Confirm Save*.
++
+You have completed the steps for re-creating the sample data map.
+
+*Next steps:*
+
+* Continue with this tutorial and use your map in a Kibana Dashboard.
```

Comment:
How about making "use your map in a Kibana dashboard" a link to the next section in the tutorial.

Note:  use a lower case d for dashboard.